### PR TITLE
pipewire: update to 0.3.72

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/pipewire/template [hunk 1]
```diff
@@ -34,11 +35,13 @@ license="MIT"
...
 distfiles="https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/${version}/pipewire-${version}.tar.gz"
-checksum=070dcf83c514903d603351921c7829014c8d9162c49ae5a043290c920f6a6363
+checksum=eb11d03e7c700e0d9204624fd3116f165f104c37eff30e93e7ac6df763e93829
 make_dirs="/var/lib/pipewire 0755 _pipewire _pipewire"
 system_accounts="_pipewire"
 
-build_options="sdl2"
+build_options="ffado sdl2"
+desc_option_ffado="Enable support for FireWire based audio devices"
+desc_option_sdl2="Enable support for SDL2"
```
Comment (classabbyamp) [inline srcpkgs/pipewire/template:44]: this already exists in common/options.description

File: srcpkgs/pipewire/template [hunk 2]
```diff
@@ -16,14 +16,15 @@ configure_args="
  -Dbluez5-codec-lc3plus=disabled
  -Dsession-managers=[]
  -Drlimits-match=@_pipewire
+ -Dlibffado=$(vopt_if ffado enabled disabled)
```
Comment (classabbyamp) [inline srcpkgs/pipewire/template:19]:
```suggestion
 $(vopt_bool ffado libffado)
```

Comment (cinerea0) [reply srcpkgs/pipewire/template:19]: That option specifically looks for "enabled", "disabled", or "auto" (see error below), which I don't think any of the `vopt_` shortcuts provide.

`meson.build:1:0: ERROR: Value "false" (of type "string") for combo option "Enable code that depends on libffado" is not one of the choices. Possible choices are (as string): "enabled", "disabled", "auto".`

Comment (classabbyamp) [reply srcpkgs/pipewire/template:19]: hm strange, meson is usually good about that

**Pull request merged**